### ISSUE TITLE
Test failed: GC\LargeMemory\API\gc\gettotalmemory\gettotalmemory.cmd

### ISSUE THREAD

**Issue by VincentBu** [dotnet/runtime#13722]
**Job:**
[coreclr-outerloop #20191103.8 Run Test Pri1 Windows_NT x64 checked](https://dev.azure.com/dnceng/public/_build/results?buildId=414072)

**Detail:**
https://helix.dot.net/api/2019-06-17/jobs/dd798cf9-2411-4416-b770-49fd6952f343/workitems/PayloadGroup0/console

**OS & Arch:**
windows x64

**Mode:**
not set

**Note:**
This failure **doesn't repro** on our windows x64 machine

**Log:**
```
GC\LargeMemory\API\gc\gettotalmemory\gettotalmemory.cmd [FAIL]
      
      Return code:      1
      Raw output file:      C:\dotnetbuild\work\dd798cf9-2411-4416-b770-49fd6952f343\Work\5e170a39-dbea-4b83-8c6f-c75044f3fe3f\Exec\GC\LargeMemory\Reports\GC.LargeMemory\API\gc\gettotalmemory\gettotalmemory.output.txt
      Raw output:
      BEGIN EXECUTION
       "C:\dotnetbuild\work\dd798cf9-2411-4416-b770-49fd6952f343\Payload\corerun.exe" gettotalmemory.dll 2048 
      Large Memory Machine required
      Test failed
      Expected: 100
      Actual: 0
      END EXECUTION - FAILED
      FAILED
      Test Harness Exitcode is : 1
```

**Comment by mangod9** [comment]
I am guessing this test fails on specific helix machines which are low on memory with `Large Memory Machine required`.. hence the failure is intermittent. @Maoni0 / @cshung ?

**Comment by Maoni0** [comment]
most likely a dup of [#13551](https://github.com/dotnet/runtime/issues/13551) which @cshung fixed with [#32545](https://github.com/dotnet/runtime/pull/32545)

**Comment by mangod9** [comment]
Ok makes sense closing this for now, since it hasnt reproed since that fix.
